doc/compiler/jitserver/Build.md

```diff
@@ -128,6 +118,26 @@ bash get_source.sh -openj9-repo=https://github.com/<Your GitHub UserID>/openj9.g
 ```
 See https://www.eclipse.org/openj9/oj9_build.html for more detail. The only difference is you need to check out the JITServer code instead of upstream OpenJ9.
 
+You can test it by running the `java -version` command.
```

Comment:
Let's change the phrasing slightly so that we don't give the impression that the entire testing is done with `java -version`. Maybe say
`You can test the success of the build process with "java -version" command like so:...`

Reply:
Actually since this is about JITServer I would use "./bin/jitserver" and "./bin/java -XX:+UseJITServer -version"